SPECS/reaper/CVE-2024-6484.patch

```diff
@@ -0,0 +1,69 @@
+From 620b02881ae264fe9cffb008626f1c11de4447d4 Mon Sep 17 00:00:00 2001
+From: Sergey Odinokov <odinserj@hangfire.io>
+Date: Tue, 18 Mar 2025 12:00:44 +0700
+Subject: [PATCH] Fix CVE-2024-6484 vulnerability by disabling further event
+ handling
+
+ Link: https://github.com/odinserj/bootstrap/commit/0ea568be7ff0c1f72a693f5d782277a9e9872077
```

Comment:
Please use "Upstream" keyword to specify that it is the upstream reference of the backported patch